airbyte-db/lib/src/test/java/io/airbyte/db/check/impl/JobsDatabaseAvailabilityCheckTest.java

```diff
@@ -28,10 +28,11 @@ void checkDatabaseAvailability() {
 
   @Test
   void checkDatabaseAvailabilityTimeout() {
-    final DSLContext dslContext = mock(DSLContext.class);
-    when(dslContext.fetchExists(any(Select.class))).thenThrow(new DataAccessException("test"));
-    final var check = new JobsDatabaseAvailabilityCheck(dslContext, TIMEOUT_MS);
-    Assertions.assertThrows(DatabaseCheckException.class, () -> check.check());
+    try (final DSLContext dslContext = mock(DSLContext.class)) {
```

Comment:
Just curious - why add a try with resources here? Is it actually at all useful for a mocked object to be closed?

Reply:
> Just curious - why add a try with resources here? Is it actually at all useful for a mocked object to be closed?

@lmossman It has no functional value, but satisfies PMD/SpotBugs which only sees the type on the left of the assignment has not been autoclosed.